webapp/src/carousel.tsx

```diff
@@ -100,10 +100,10 @@ export class Carousel extends data.Component<ICarouselProps, ICarouselState> {
                     }
                 </div>
             </div>
-            <span role="button" className={"carouselarrow right aligned" + (rightDisabled ? " arrowdisabled" : "")} aria-label={lf("See more")}
-                tabIndex={rightDisabled ? -1 : 0} onClick={this.onRightArrowClick} onKeyDown={sui.fireClickOnEnter} ref={this.handleArrowRefs}>
+            {!rightDisabled && <span role="button" className={"carouselarrow right aligned"} tabIndex={0}
```

Comment:
If both left and right are not disabled (like if you're in the middle of a long carousel) do the tab indices need to be different?

Reply:
tab index 0 says to use regular tab order (based off html ordering); only need to be unique if >= 1